Changelog Aug 14 2025.md (#4361)
timescale/docs: about/changelog.md

All the latest features and updates to $CLOUD_LONG.

## Cross-region backups, PostgreSQL options, and onboarding
<Label type="date">August 14, 2025</Label>

### Cross-region backups

> You can now store backups in a different region than your service, which improves resilience and helps meet enterprise compliance requirements. Cross‑region backups are available on our Enterprise plan for free at launch; usage‑based billing may be introduced later. For full details, please [see the docs](https://docs.tigerdata.com/use-timescale/latest/backup-restore/#enable-cross-region-backup).

Reviewer: ```suggestion
You can now store backups in a different region than your service. This improves resilience and helps meet enterprise compliance requirements. Cross‑region backups are currently available for free on our Enterprise plan; usage‑based billing may be introduced later. For full details, [see the docs](https://docs.tigerdata.com/use-timescale/latest/backup-restore/#enable-cross-region-backup).
```

### Standard PostgreSQL instructions for onboarding

> We have added basic instructions for INSERT, UPDATE, DELETE commands to the Tiger Cloud console.  It's now shown as an option in the Import Data page.

Reviewer: ```suggestion
We have added basic instructions for INSERT, UPDATE, DELETE commands to Tiger Cloud Console.  This information is now shown as an option in the Import Data page.
```
